ravenframework/CodeInterfaceClasses/Dymola/DymolaInterface.py

```diff
@@ -318,6 +318,25 @@ def createNewInput(self, currentInputFiles, oriInputFiles, samplerType, **Kwargs
           "Parameter %s does not exist or is not formatted as expected "
           "in %s." % (name, originalPath))
 
+    ## set fileName parameter which points Dymola to vector table
+    if bool(vectorsToPass):
+      initVarIndex = 0
+      fileNameInitIndex = 0
+      initDescripIndex = 0
+      splitLines = text.splitlines()
+      for i, line in enumerate(splitLines):  #find location of path
+        if "char initialName" in line:
+          initVarIndex = i
+        if line == 'fileName' and initVarIndex > 0 and fileNameInitIndex == 0:
+          fileNameInitIndex = i
+        if "char initialDescription" in line:
+          initDescripIndex = i
+      if fileNameInitIndex > 0:
+        filePathIndex = initDescripIndex + fileNameInitIndex - initVarIndex #index of path
+        text = text.replace(splitLines[filePathIndex], currentInputFiles[indexVect].getAbsFile().replace(os.sep, '/'))
```

Comment:
could you explain why 'os.sep' is replaced with '/'

Reply:
I tested the code without .replace(os.sep, '/') and it ran well without any issues. I will change the code line accordingly.

Reply:
And I will add a new regression test for this as well.